.github/workflows/build.yml

```diff
@@ -25,8 +25,6 @@ jobs:
           # distributions and container images.
 
           # linux x64 glibc
-          - os: ubuntu-20.04
```

Comment:
Once merged to sentry-javascript we'll adopt their v7.xx version so it's also a good time to drop node 14.